arches/app/src/components/ControlledListManager/SearchAddDelete.vue

```diff
@@ -0,0 +1,127 @@
+<script setup lang="ts">
+import arches from "arches";
+
+import Button from "primevue/button";
+import InputText from "primevue/inputtext";
+import SplitButton from "primevue/splitbutton";
+
+const buttonGreen = "#10b981";
+const buttonPink = "#ed7979";
+
+const props: {
+    addLabel: string;
+    createItem: () => Promise<void>;
+    deleteItems: () => Promise<void>;
+    deleteLabel: string;
+    deleteLabelPlural: string;
+    numberToDelete: number;
+} = defineProps([
+    "addLabel",
+    "createItem",
+    "deleteItems",
+    "deleteLabel",
+    "deleteLabelPlural",
+    "numberToDelete",
+]);
+
+const searchValue = defineModel<string>({ required: true });
+
+const clearSearch = () => {
```

Comment:
could this just be inlined since it's only used once?

Reply:
It's used for both @click and @keydown